docs-devsite/auth.md

```diff
@@ -720,6 +722,7 @@ A `Promise` that resolves once the persistence change has completed
 setPersistence(auth, browserSessionPersistence);
 
 ```
+This method has no effect in Node.js.
```

Comment:
@sam-gc @renkelvin is this unsupported on Node?

Reply:
Unless I'm reading this wrong (there's an explainer comment above this line as well):
https://github.com/firebase/firebase-js-sdk/blob/92e2279a299dbe88e01bf7ed3da4be42d4de86c9/packages/auth/src/platform_node/index.ts#L113

Reply:
My bad, i thought this was for signInAnonymously (the function documented below). Resolving.

Reply:
Actually, can we move this to line 698 , to be consistent with other callouts that are before the "Signature" section.